connection.go

```diff
@@ -143,6 +143,12 @@ type Connection struct {
 
 var _ = Connector(&Connection{}) // Check compatibility with connector interface.
 
+// ConnectedObject is an interface that provides the info about a Connection
+// the object belongs to.
+type ConnectedObject interface {
```

Comment:
What do you think about just `Connected`? It's shorter acceptable in my taste.

Reply:
It's a pity, maybe then do this?

[conn.patch.txt](https://github.com/tarantool/go-tarantool/files/9100698/conn.patch.txt)

Reply:
Yes, patched this way.